apps/files/src/views/folderTree.ts

```diff
@@ -151,7 +151,7 @@ const registerTreeRoot = () => {
 	Navigation.register(new View({
 		id: folderTreeId,
 
-		name: t('files', 'All folders'),
+		name: t('files', 'Folder tree structure'),
```

Comment:
"tree structure" is rather technical and long.

```suggestion
		name: t('files', 'Folder tree'),
```